### PR TITLE
Issue 4 user delete entry

### PR DESCRIPTION
**Pull request by jonahp001**
Kapture 2023-02-16 at 14 50 26

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -170,7 +180,39 @@ $ulElement.addEventListener('click', function (event) {
...
+    if (parseInt($liInUl[j].getAttribute('data-entry-id')) === data.editing.entryID) {
+      $liInUl[j].remove();
+    }
+  }
+
+  if (data.entries.length === 0) {
+    toggleNoEntries();
+  }
+  $modalBg.setAttribute('class', 'modal-bg hidden');
+  viewSwap('entries');
```
Comment (robgmerrill) [inline js/main.js:217]: You will need to decrement data.entries.entryId by one. If you look in your gif you will see that this number keeps growing as you delete items and add items.

**Review by robgmerrill — CHANGES_REQUESTED** [review]
Very close! I've left a comment for what you should do at the end of the callback function on your confirm button.

**Comment by jonahp001** [comment]
Kapture 2023-02-18 at 19 42 55

fixed!!!! :)

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**